.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+---
+- repo: https://github.com/pre-commit/mirrors-prettier
+  rev: "" # Use the sha or tag you want to point at
```

Comment:
Nitpick: can we pin it to a tag? Just to avoid accidental reformatting at some point in the future.